### PR TITLE
Adding new dashboard images and reordering widgets

### LINKED ISSUE #716: [doc] Figure 1.2:  Ceph Dashboard Home Page
### Type of report
   - [X] bug
   - [ ] feature request

### Source Document
Figure&nbsp;1.2:  Ceph Dashboard Home Page :

https://documentation.suse.com/ses/7/single-html/ses-admin/#id-1.4.3.2.6.2.9

### Summary
The screen shot of the Dashboard landing page needs to be replaced. There have been some visual changes/improvements that are not reflected in the current screen shot:

Current screenshot:

dashboard_homepage

Actual look:

Screenshot from 2020-10-22 13-43-22

### Steps To Reproduce
Open the Dashboard landing page in the current SES 7 milestone, compare the look with the one depicted in the docs.

### Expected Results

The screen shot included in the docs should resemble the actual look.

### Notes

Please replace the current screen shot with a newer one. Feel free to use the one attached to this issue.

### PR DESCRIPTION
**Pull request by asettle**
Closes #716

### CODE REVIEW

**Review by tbazant — APPROVED** [review]
lgtm

File: xml/admin_gui_dashboard.xml [hunk 1]
```diff
@@ -392,123 +392,123 @@
...
        </para>
       </listitem>
      </varlistentry>
      <varlistentry>
-      <term><guimenu>Raw Capacity</guimenu></term>
+      <term><guimenu>Client Throughput</guimenu></term>
       <listitem>
        <para>
-        Shows the ratio of used and available raw storage capacity.
+        The sum of clients' read and write operations per second.
```
Comment (bk201) [inline xml/admin_gui_dashboard.xml:485]: (Not incurred by this PR). The definition here sounds like IOPS.

Comment (asettle) [reply xml/admin_gui_dashboard.xml:485]: I'm not sure what you need to be changed here?

Comment (LenzGr) [reply xml/admin_gui_dashboard.xml:485]: The original description was wrong already. "The sum of clients' read and write operations per second." was used for both the description of the "Client IOPS" and "Client Throughput" widget.

The description for Client Throughput should read:
```suggestion
        The amount of data transferred to/from Ceph clients in multiples of bytes per second.
```

**Comment by LenzGr** [comment]
Looking good, thank you! Note that the order of widgets also changed slightly, also in the top "Status" part:

Status Widgets:
Cluster Status
Hosts
Monitors
OSDs
Managers
Object Gateways
Metadata Servers
iSCSI Gateways

Capacity Widgets:
Raw Capacity
Objects
PG Status
Pools
PGs per OSD

Performance Widgets:
Client Read/Write
Client Throughput
Recovery Throughput
Scrubbing

**Comment by asettle** [comment]
Screenshot from 2020-10-22 14-56-37
Status widgets

**Comment by asettle** [comment]
Screenshot from 2020-10-22 14-57-08
Capacity widgets

**Comment by asettle** [comment]
Screenshot from 2020-10-22 14-57-31
performance widgets

**Comment by LenzGr** [comment]
Some minor nits:
- The "Manager Daemons" widget is now labeled "Managers"
- "iSCSI Gateway" is now "iSCSI Gateways"
- "Client IOPS" is now "Client Read/Write"

**Review by LenzGr — APPROVED** [review]
LGTM! Thanks for the swift fix, @asettle

**Pull request merged**